apparelo/apparelo/doctype/dyeing/dyeing.py

```diff
@@ -17,6 +17,18 @@ class Dyeing(Document):
 	def on_submit(self):
 		create_item_template()
 		create_item_attribute()
+		self.validate_mappings()
+
+	def validate_mappings(self):
+		for colour_shade_mapping in self.colour_shade_mapping:
+			found = False
+			for colours in self.colours:
+				if colours.colour == colour_shade_mapping.colour:
+					found = True
+			if found:
+				continue 
+			else:
+				frappe.throw(_(f'The colour {colour_shade_mapping.colour} entered in colour shade mapping table was not found in colours table.'))
```

Comment:
remove this validation as discussed....

Reply:
The above-requested changes are updated in this commit https://github.com/aerele/apparelo/pull/187/commits/09c97c4f02068b810d169d4c82f9ebfa16d1a42a

Reply:
Here, I updated the code to handle location-wise deliver later items for print format https://github.com/aerele/apparelo/pull/187/commits/03c7d735eb0ae63e15d79969331eed31f5ee0aeb